docs/manual/common/concepts/IntAndExtComm.md

```diff
@@ -0,0 +1,7 @@
+#Internal and external communication
+
+Lagom promotes use of asynchronous communication without preventing use of synchronous communication where necessary. For example, as mentioned [[previously|PolyglotSystems]], legacy application components might use synchronous communication that cannot be changed.  When choosing the mode of communication for your microservices, keep in mind inter-service communication as well as external communication. 
```

Comment:
`as mentioned [[previously|PolyglotSystems]]` <-- same comment I made earlier.

How about: `as mentioned in [[Polyglot systems with Lagom |PolyglotSystems]]` ?